docs/source/standard.md

```diff
@@ -39,67 +39,92 @@ What the TRE organisation does to ensure information risk is measured and manage
+| You have a process in place to monitor changes to any legal, ethical and contractual requirements, and to update your policies accordingly | | Mandatory |
 
 ### 1.3 Quality management
 
-The ability of the TRE organisation to measure and control quality of processes, documentation and outputs.
+*The ability of the TRE organisation to measure and control quality of processes, documentation and outputs.*
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+#### Document management
 
-#### 1.4 Audit and reporting
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| All policies & standard operating procedures relevant to the TRE organisation are controlled | This may include measures like restricting edit access to relevant documents, and recording acceptance of policies for all TRE organisation members | Mandatory |
 
-The ability of the TRE organisation to adhere to stated processes and external standards.
+#### Issue management
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have a clear process in place for addressing activity within your TRE organisation that deviates from your policies and standard operating procedures | This can include measures like triage analysis and a process for updating policies | Mandatory |
+| You have methods in place to record progress in resolving issues with, and deviations against, your policies | | Mandatory |
 
-### 1.5 Risk management
+### 1.4 Risk management
 
-The ability of the TRE organisation to measure, forecast and evaluate risks to information.
+*The ability of the TRE organisation to measure, forecast and evaluate risks to information.*
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+#### Risk assessment
+
+| Statement | Guidance | Mandatory Status |
+| --------- | -------- | ---------------- |
+| You have a way to score risk to understand the underlying severity | You have a risk assesment methodology for scoring risks on multiple axes such as impact and likelihood | Mandatory |
+| You have a process for treating risk using additional controls | Risks can be reduced to a level which brings it within agreed levels of appetite | Mandatory |
+| You have an understanding of risk appe*tite | You have an understanding of risk appetite, ownership of risk and ability to accept risk which falls outside of the appetite should that become necessary | Mandatory |
+| You carry out a Data Protection Impact Assessment (DPIA) for all projects requiring a TRE that are working with personal data | a DPIA is a process designed to identify risks arising out of the processing of personal data and to minimise these risks as far and as early as possible | Mandatory |
```

Comment:
This is very UK-centric. Do we want to be? If not, how about a (non-capitalised) "data processing assessment"? Also, do we only care about personal data? How about "sensitive data" instead?

Reply:
This was my thought too, but decided not to open that can of worms! I think it'd be great if SATRE could be relevant across Europe, and eventually beyond.